ansible/sample_production/group_vars/all

```diff
@@ -0,0 +1,21 @@
+---
+api_id: "{{ ansible_env.API_CLIENT}}"
+key_file: "{{ ansible_env.PRIV_KEY_FILE}}"
+#api_id: "pure1:apikey:123456789"
+#key_file: "/home/user/key.pem"
+
+# storage service variables
+storage_service_name: ss_name
+
+# storage class variables
+storage_class_name: sc_name
```

Comment:
This is ok for now. But could you think about how to structure the vars file so that a user can specify multiple storage services, and within each storage service, multiple storage classes? E.g. in [policy.yaml](https://github.com/pure-aferrario/fusion-client-setup/blob/main/python/config/policy.yaml)